lib/getopt.c

```diff
@@ -29,7 +29,9 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#ifndef _WIN32
```

Comment:
Cygwin and also MinGW-w64 have `unistd.h` so this should be `#ifdef _MSC_VER`.

Reply:
Fixed in a new commit.